xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/pom.xml

```diff
@@ -61,6 +61,12 @@
       <artifactId>xwiki-platform-template-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-activitystream-api</artifactId>
+      <version>${project.version}</version>
```

Comment:
is this needed at build time or runtime? It's strange that it was missing before if it's needed at build time.